chore(CHANGELOG): update to v1.0.0-alpha.0

barmac opened the pull request:
This is a concise update of the changelog which will prepare us for the first alpha. It will be followed by updated examples and the blog post which we should have ready before the first stable release.

nikku commented on CHANGELOG.md:
```diff
@@ -6,6 +6,16 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
+## 1.0.0-alpha.0
+
+* `FEAT`: rewrite project to new, [Preact](https://github.com/preactjs/preact)-based architecture
```
Add fundamental rework of UI.
Add removal of tabs.
Add dirty markers being introduced.

nikku commented on CHANGELOG.md:
```diff
@@ -6,6 +6,16 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
+
+### Breaking Changes
+
+* `PropertiesProvider#getTabs` is no longer used. Migrate to the new `PropertiesProvider#getGroups` API instead.
```
Consider pointing to an example how to migrate (before -> after).

That could be a commit on the `MagicProperties` example, if need be.

barmac commented:
@nikku Please check now. I added more details on the changes and linked the example.

Also note that https://github.com/bpmn-io/bpmn-js-properties-panel/pull/488/commits/dddaca219edb24f227c57859b5e876dba62d0cac fixes a problem I introduced via https://github.com/bpmn-io/bpmn-js-properties-panel/commit/319aaf9c72e56a93762a5fbe63245d0dd4a86c38 which caused rollup to bundle all of the dependencies 🙈 . Now it doesn't bundle anything apart from the devDependencies if necessary.

nikku reviewed (approved):
Very nice!

nikku commented:
Likely unrelated but [CI is failing](https://github.com/bpmn-io/bpmn-js-properties-panel/runs/4548115179?check_suite_focus=true#step:6:1730).
